<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView_gameView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>911</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_informationsView">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_optionsView">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>180</y>
     <width>241</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_buttonView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>911</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ptr_textBrowserLog">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>600</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
